api/feedback/v1/feedback.proto

```diff
@@ -93,29 +93,24 @@ message FeedbackMetadata {
   // extra info on the feedback (i.e. the survey question, the rating options, the feedback origin, etc.)
   Origin origin = 1 [ (validate.rules).enum = {defined_only : true not_in : 0} ];
   Survey survey = 2 [ (validate.rules).message = {required : true} ];
-  // TODO: remove if we decide to record feedback only when a user clicks the submit button
   bool user_submitted = 3;
   string url_search_params = 4;
 }
 
 message Feedback {
-  // url path of where the feedback was submitted
-  string url_path = 1 [ (validate.rules).string = {min_bytes : 1} ];
+  // url path or workflow name that received feedback
```

Comment:
nit: Since 'General' and possibly others wont be a url path or a workflow name, should we adjust this text?

Maybe:
'url path, workflow name or area of feedback'